packages/lesswrong/components/localGroups/CommunityMap.tsx

```diff
@@ -108,7 +109,7 @@ const CommunityMap = ({ groupTerms, eventTerms, initialOpenWindows = [], center
       longitude: center.lng,
       zoom: zoom
     })
-  }, [center, zoom])
+  }, [center.lat, center.lng, zoom])
```

Comment:
Also fixed an annoying bug where the map viewport would get reset when anything on the page changed, even when the location wasn't changing.